packages/ckeditor5-table-resize/tests/tablecolumnresizer/utils.js

```diff
@@ -330,6 +338,131 @@ describe( 'TableColumnResize utils', () => {
 				expect( affectedTables.size ).to.equal( 0 );
 			} );
 		} );
+
+		describe( 'should not crash if no affected table has been found', () => {
+			let model, differ, editor, editorElement, linkCommand, unlinkCommand, highlightCommand;
+
+			beforeEach( async () => {
+				editorElement = document.createElement( 'div' );
+				document.body.appendChild( editorElement );
+
+				editor = await ClassicEditor.create( editorElement, Object.assign( {}, {
+					plugins: [ Table, Paragraph, LinkEditing, HighlightEditing, Bold ]
+				} ) );
```

Comment:
The syntax here (I guess copied from the `createEditor()` function) is useful if we want to allow the config extensions, but here we can pass the config object directly.